frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { screen } from '@testing-library/react';
+import { render } from 'utils/testRenderer';
+import { StrategyItemContainer } from './StrategyItemContainer';
+import { IFeatureStrategy } from 'interfaces/strategy';
+import { testServerRoute, testServerSetup } from 'utils/testServer';
+import { UIProviderContainer } from '../../providers/UIProvider/UIProviderContainer';
+
+const server = testServerSetup();
+testServerRoute(server, '/api/admin/ui-config', {
+    flags: { strategyImprovements: true },
+});
+
+test('should render authorization error on missing old password', async () => {
```

Comment:
Is this test description accurate? What here relates to a password?

Reply:
good catch of copy paste :)